Makefile

```diff
@@ -602,7 +602,7 @@ generate: $(TAGS_PREREQ)
 	@CC= GOOS= GOARCH= $(GO) generate -tags '$(TAGS)' $(GO_PACKAGES)
 
 $(EXECUTABLE): $(GO_SOURCES) $(TAGS_PREREQ)
-	CGO_CFLAGS="$(CGO_CFLAGS)" $(GO) build $(GOFLAGS) $(EXTRA_GOFLAGS) -tags '$(TAGS)' -ldflags '-s -w $(LDFLAGS)' -o $@
+	CGO_CFLAGS="$(CGO_CFLAGS)" $(GO) build -buildvcs=false $(GOFLAGS) $(EXTRA_GOFLAGS) -tags '$(TAGS)' -ldflags '-s -w $(LDFLAGS)' -o $@
```

Comment:
Why aren't you just setting EXTRA_GOFLAGS to use this flag for your specific need. Most people building Gitea do not need this flag and will not need it?

Reply:
Even if the reasons above don't justify the use of this flag, what value does this flag remove from other users? It just resets the build to behave as pre-1.18.